themes/palenight-italic.json

```diff
@@ -1181,53 +1181,95 @@
       }
     },
     {
-      "name": "Markdown Headings",
-      "scope": "markup.heading.markdown",
+      "name": "Markup Headings",
+      "scope": "markup.heading",
       "settings": {
         "foreground": "#82b1ff"
       }
     },
     {
-      "name": "Markdown Italics",
-      "scope": "markup.italic.markdown",
+      "name": "Markup Italics",
+      "scope": "markup.italic",
       "settings": {
         "foreground": "#c792ea",
         "fontStyle": "italic"
       }
     },
     {
-      "name": "Markdown Bold",
-      "scope": "markup.bold.markdown",
+      "name": "Markup Bold",
+      "scope": "markup.bold",
       "settings": {
         "foreground": "#ffcb6b",
         "fontStyle": "bold"
       }
     },
     {
-      "name": "Markdown Quote + others",
-      "scope": "markup.quote.markdown",
+      "name": "Markup Quote + others",
+      "scope": "markup.quote",
       "settings": {
         "foreground": "#697098",
         "fontStyle": "italic"
       }
     },
     {
-      "name": "Markdown Raw Code + others",
-      "scope": "markup.inline.raw.markdown",
+      "name": "Markup Raw Code + others",
+      "scope": "markup.inline.raw",
       "settings": {
         "foreground": "#80CBC4"
       }
     },
     {
-      "name": "Markdown Links",
+      "name": "Markup Links",
       "scope": [
-        "markup.underline.link.markdown",
-        "markup.underline.link.image.markdown"
+        "markup.underline.link",
+        "markup.underline.link.image"
       ],
       "settings": {
         "foreground": "#ff869a"
       }
     },
+    {
+      "name": "Markup Fenced Code",
+      "scope": [
+        "fenced_code.block.language"
+      ],
+      "settings": {
+        "foreground": "#80CBC4"
+      }
+    },
+    {
+      "name": "Markup Attributes",
+      "scope": [
+        "markup.meta.attribute-list"
+      ],
+      "settings": {
+        "foreground": "#a9c77d"
+      }
+    },
+    {
+      "name": "Markup Admonitions",
+      "scope": "markup.admonition",
```

Comment:
Nice, TIL! Curious question: Does GFM support admonitions?

Reply:
GFM does not, sorry ;-)
RestructuredText does support them, but their VS Code extension doesn't tag their content as such, so we can't style them. For now, it seems this scope is only for asciidoc